omniscidb/Tests/IntelGPUEnablingTest.cpp

```diff
@@ -445,6 +445,26 @@ TEST_F(AggregationTest, NullHandling) {
   c("SELECT COUNT(*) FROM test WHERE ofq >= 0 OR ofq IS NULL;", g_dt);
 }
 
+class OverflowTest : public ExecuteTestBase, public ::testing::Test {};
+
+TEST_F(OverflowTest, OverflowAndUnderFlow) {
+  c("SELECT dd * ufd FROM test;", g_dt);
+  c("SELECT dd * -1 FROM test;", g_dt);
+  c("SELECT -1 * dd FROM test;", g_dt);
+  c("SELECT ofq * -1 FROM test;", g_dt);
+  c("SELECT 1 * ofq FROM test;", g_dt);
+  c("SELECT 566 * 244 FROM test;", g_dt);
+  EXPECT_THROW(c("SELECT ofd * ofd FROM test;", g_dt), std::runtime_error);
```

Comment:
This can be a more reliable test if you don't run the comparison. The `runtime_error` may come from sqlite or the comparison itself, so we'll not catch an issue with hdk not detecting an overflow.